### ISSUE TITLE
Enhancement: Default Group 

### ISSUE THREAD

**Issue by MordyT** [Ylianst/MeshCentral#1409]
An enhancement where you can specify a default group for new users. Could be used with a SSO option to say the users from SSO  automatically get group 1, new users from mesh DB get group 2, etc.

Something like:
"saml": {
          "defaultgroup": "Servers",    <----------
          "entityid": "meshcentral",
          "idpurl": "https://server/saml2",
          "cert": "saml.pem"
        }

**Comment by Ylianst** [comment]
When you indicate a default group, is this "User Group" or "Device Group" or "Administrative Realm"?

**Comment by MordyT** [comment]
I wish then when a user comes in via SAML SSO, a device group is assigned to them automatically. The idea being that I can control SSO and which users can use it - so if a user can use SSO, I want them to have access to certain things out the door,

**Comment by Ylianst** [comment]
Ok, got it. Will add that.

**Comment by Ylianst** [comment]
Published MeshCentral v0.5.46 with support for this. You can add this line in the "domain" section of the config.json, or in the "saml" section.

```
"newAccountsUserGroups": [ "ugrp//nbkB4mWQeXXTi8xbLdIxAom7zynGYAsEfFB$a7XR1ooQMH3E94r8Q6uZrMm@yEdf" ]
```

This will auto-join users to the specified user group. Of course, you can give this user group rights over device groups or individual devices, so this is the most flexible way.

To see all user groups and identifiers on your server, run this:

```
node node_modules/meshcentral --showusergroups
```

Feedback appreciated.

**Comment by MordyT** [comment]
This worked perfectly. Thanks!
